fetch-utils.js

```diff
@@ -15,7 +15,7 @@ export function checkAuth() {
 
 export function redirectIfLoggedIn() {
     if (getUser()) {
```

Comment:
Really like the simplicity of this redirect. You can also try to use a ternary which would like like ' getUser() ? location.replace('./mylist') : window.location.href = '../'